inference-engine/tests/functional/plugin/shared/src/single_layer_tests/detection_output.cpp

```diff
@@ -0,0 +1,151 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <tuple>
+#include <vector>
+
+#include "ngraph_functions/builders.hpp"
+#include "common_test_utils/data_utils.hpp"
+#include "functional_test_utils/layer_test_utils.hpp"
+#include "single_layer_tests/detection_output.hpp"
+
+namespace LayerTestsDefinitions {
+
+const float nmsThreshold = 0.5f;
+const float confidenceThreshold = 0.3f;
+const float objectnessScore = 0.4f;
+
+std::string DetectionOutputLayerTest::getTestCaseName(testing::TestParamInfo<DetectionOutputParams> obj) {
+    DetectionOutputAttributes commonAttrs;
+    ParamsWhichSizeDepends specificAttrs;
+    size_t batch;
+    std::string targetDevice;
+    std::tie(commonAttrs, specificAttrs, batch, targetDevice) = obj.param;
+
+    ngraph::op::DetectionOutputAttrs attrs;
+    std::tie(attrs.num_classes, attrs.top_k, attrs.keep_top_k, attrs.code_type, attrs.clip_after_nms, attrs.clip_before_nms,
+             attrs.decrease_label_id) = commonAttrs;
+
+    const size_t numInputs = 5;
+    std::vector<InferenceEngine::SizeVector> inShapes(numInputs);
+    std::tie(attrs.variance_encoded_in_target, attrs.share_location, attrs.normalized, attrs.input_height, attrs.input_width,
+             inShapes[idxLocation], inShapes[idxConfidence], inShapes[idxPriors], inShapes[idxArmConfidence], inShapes[idxArmLocation]) = specificAttrs;
+
+    for (size_t i = 0; i < inShapes.size(); i++) {
+        if (inShapes[i].empty())
+            break;
```

Comment:
What is reason? And if there is 2 empty inputs?

Reply:
fixed